ImperatorToCK3/Source/CK3Outputter/CK3WorldOutputter.cpp

```diff
@@ -16,7 +16,7 @@ void createModFolder(const std::string& outputName);
 
 void CK3World::outputWorld(const CK3World::World& world)
 {
-	std::string outputName = world.getOutputModName();
+	const auto outputName = world.getOutputModName();
```

Comment:
Same thing here, if you're returning a reference up there, grab that here and operate on in. So const auto&.